.circleci/config.yml

```diff
@@ -41,21 +47,18 @@ commands:
     description: "Add changes to SVN"
     steps:
       - run:
-          command: if [[ ! -z $(svn st | grep ^\!) ]]; then svn st | grep ^! | awk '{print " --force "$2}' | xargs -0r svn rm; fi
```

Comment:
This is the main fix. It doesn't attempt to add files that don't exist anymore, like `svgxuse.js`.